### PR TITLE
zeromq 4.3.0

### PR DESCRIPTION
**Pull request by igas**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
This is hit by the infamous:

```
In file included from src/v2_decoder.cpp:33:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/cmath:313:9: error: no member named 'signbit' in the global namespace
```

File: Formula/zeromq.rb [hunk 1]
```diff
@@ -20,35 +20,22 @@ class Zeromq < Formula
     depends_on "libtool" => :build
   end
 
-  option "with-libpgm", "Build with PGM extension"
-  option "with-norm", "Build with NORM extension"
-  option "with-drafts", "Build and install draft classes and methods"
```
Comment (Mystfit) [inline Formula/zeromq.rb]: You've removed the --with-drafts option which means we won't be able to build with the draft api enabled. Could we keep this option or add --enable-drafts to the configure step by default?

Comment (SMillerDev) [reply Formula/zeromq.rb]: Our analytics show that 44 people out of ~44.000 used it in the past year. I'd say the api isn't really needed.

Comment (Mystfit) [reply Formula/zeromq.rb]: That's a shame. I'm one of those 44 people so I guess I won't be able to use Homebrew to obtain ZeroMQ.

Comment (igas) [reply Formula/zeromq.rb]: You still can use homebrew, but suggested path is to make your own tap. Here are the docs on how to do it https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

For core brew repo it's hard to test all the possible combination of flags on CI (and we don't test them) as a consequence often flags broken after version upgrades and consequentially for the people like you we bring broken upgrade. While if it critical software for you, you know when to upgrade and when flags work.

As @fxcoudert mentioned before, there is more info in #31510 with the reasoning behind this.

Comment (Mystfit) [reply Formula/zeromq.rb]: Thanks @igas. I'll create a tap preserving the draft flags until ZeroMQ gets around to marking the features I use as stable.

**Comment by scpeters** [comment]
@BrewTestBot test this please

**Comment by scpeters** [comment]
I can reproduce the `signbit` build error on high sierra with the current version of the formula (4.2.5)

**Comment by igas** [comment]
@BrewTestBot test this please

**Comment by aimileus** [comment]
It seems to be a problem with the options added by superenv. If the build system calls clang directly by adding `ENV["CXX"] = "/usr/bin/clang++"`, the build completes fine.

**Comment by aimileus** [comment]
The workaround works with `ENV.delete("HOMEBREW_SDKROOT")`.

**Comment by igas** [comment]
Thank you @aimileus!

**Comment by bluca** [comment]
@igas hi, could you please update the formula to the newly released 4.3.1 as soon as you have time? Unfortunately I do not have OSX to do it myself. The new version fixes a remote execution vulnerability: https://github.com/zeromq/libzmq/releases/tag/v4.3.1
Thanks!

**Comment by aimileus** [comment]
I opened a new PR at #35940

**Pull request closed without merging**